pkg/perf/map.go

```diff
@@ -0,0 +1,31 @@
+package perf
+
+import (
+	"errors"
+	"sort"
+)
+
+var (
+	ErrNoSymbolFound = errors.New("no symbol found")
+)
+
+type MapAddr struct {
+	Start  uint64
+	End    uint64
+	Symbol string
+}
+
+type Map struct {
+	addrs []MapAddr
+}
+
+func (p *Map) Lookup(addr uint64) (string, error) {
```

Comment:
This or `Resolve` can be an interface for similar components like `ksym` and `vdso` components.

Reply:
I think I just kept whatever it was.

Reply:
👍 We can eventually converge on an interface, and we should. No need to do it in this PR.